test/test_ops.py

```diff
@@ -538,5 +538,15 @@ def script_func(x_, offset_, weight_, bias_, stride_, pad_, dilation_):
                   (x, offset, weight, bias), nondet_tol=1e-5)
 
 
+class FrozenBNTester(unittest.TestCase):
+    def test_frozenbatchnorm2d_repr(self):
+        num_features = 32
+        t = ops.misc.FrozenBatchNorm2d(num_features)
+
+        # Check integrity of object __repr__ attribute
+        expected_string = 'FrozenBatchNorm2d({0})'.format(num_features)
```

Comment:
Why use f strings down below but `.format` here? Wouldn't we want to be consistent in what type of string formatting we are using?

Reply:
My bad, I added the test in a different commit and forgot about it!
Just pushed a fix